### DISCUSSION TITLE
How many 💀🦴 does the adult human body have?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1122] [Galaxy Trivia] [answered]
**Bonus:** How many bones do infants typically have?

**Comment by LeleDallas** [comment]
Adult: 206
Infants: 300

**Comment by peterbaccaro** [comment, marked as answer]
There are **typically 206 bones in the adult human body**.  The number can vary slightly due to variations in bone structure (e.g., the presence of extra ribs or accessory bones).

An **infant human body is born with approximately 270 bones**.  As a child grows, some of these bones gradually fuse together.
